internal/pkg/template/templates/environment/cf.yml

```diff
@@ -56,6 +56,9 @@ Conditions:
   HasAliases:
     !Not [!Equals [ !Ref Aliases, "" ]]
 Resources:
+{{- if .CDNConfig.EnableCDN}}
```

Comment:
Can we make CFN changes a separate PR 🙏 There are a lot to digest and by making another PRs we could potentially reduce the back-and-forth